packages/account-v2/src/components/DemoMessage/DemoMessage.tsx

```diff
@@ -4,12 +4,13 @@ import { DerivLightIcPoaLockIcon } from '@deriv/quill-icons';
 import { Button } from '@deriv-com/ui';
 import { IconWithMessage } from '../IconWithMessage';
 
-export const DemoMessage = () => {
+export const DemoMessage = ({ className }: { className?: string }) => {
     const { data: authorizeData } = useAuthorize();
     const hasRealAccount = authorizeData?.account_list?.some(account => account.is_virtual === 0);
     return (
         <IconWithMessage
             actionButton={<Button>{hasRealAccount ? 'Switch to real account' : 'Add a real account'}</Button>}
```

Comment:
Lets add a TODO to track that button functionality is to be added for click action?